pkg/gatewayserver/io/io.go

```diff
@@ -708,11 +708,12 @@ func (c *Connection) RTTStats(percentile int, t time.Time) (min, max, median, np
 func (c *Connection) Stats() (*ttnpb.GatewayConnectionStats, []string) {
 	ct := c.ConnectTime()
 	stats := &ttnpb.GatewayConnectionStats{
-		ConnectedAt: ttnpb.ProtoTimePtr(ct),
-		Protocol:    c.Frontend().Protocol(),
+		ConnectedAt:    ttnpb.ProtoTimePtr(ct),
+		DisconnectedAt: nil,
```

Comment:
No need for the explicit `nil` - it is already `nil`.

Reply:
Reflected on it the `path` already makes it explicit enough